composer.json

```diff
@@ -24,9 +24,10 @@
     },
     "require-dev": {
         "laminas/laminas-coding-standard": "^3.0",
+        "phpstan/phpstan": "^2.1",
+        "phpstan/phpstan-phpunit": "^2.0",
         "phpunit/phpunit": "^10.5.5",
-        "squizlabs/php_codesniffer": "^3.8.0",
-        "vimeo/psalm": "^6.0"
+        "squizlabs/php_codesniffer": "^3.8.0"
```

Comment:
```suggestion
```
Because it is required by https://github.com/laminas/laminas-coding-standard/blob/3.1.x/composer.json#L15